### PR TITLE
Dark mode added

### PR DESCRIPTION
**Pull request by msdeshmukh009**
**Features added**
- Dark mode for all the pages.
- Saving user preference in local storage.

**GIF**
darkmode

### CODE REVIEW

File: docsite-css/doc-site.css [hunk 1]
```diff
@@ -27,6 +27,21 @@
 }
 /*******/
 
+/*** dark-mode ***/
+.dark-mode {
+  --background-color: #16191f;
+  --text-color: rgb(181, 192, 202);
```
Comment (kshitijthatte) [inline docsite-css/doc-site.css]: Since you're not using any alpha value in rgb value of var text-color, you can use HEX for better consistency.

**Review by kshitijthatte — APPROVED** [review]
Looks Amazing! 🔥

**Review by tanveertkd — APPROVED** [review]
This is good to go! Good work Mukesh! 👍

File: docsite-css/doc-site.css (hunk 1 above)
Comment (msdeshmukh009) [reply docsite-css/doc-site.css]: Good point I will make the changes accordingly. Thanks for the review.

**Comment by msdeshmukh009** [comment]
> This is good to go! Good work Mukesh! 👍

Thanks!! and its Mahesh actually 😅

**Pull request merged**